src/components/CharactersList.js

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from "react";
+import CharacterBox from "./CharacterBox";
+import Filters from "./Filters";
+
+export default function CharactersList() {
```

Comment:
😉 https://github.com/yvlange/rick-and-morty-react-app/pull/3#discussion_r611529334